src/pages/host/Metadata.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { ApolloError } from "apollo-client";
+import { StyledLink } from "components/styles";
+import { format } from "date-fns";
+import { wordBreakCss, gray } from "components/Typography";
+import { HostMetaDataCard } from "components/HostMetaDataCard";
+import { HostQuery } from "gql/generated/types";
+import { getUiUrl } from "utils/getEnvironmentVariables";
+import styled from "@emotion/styled/macro";
+
+export const Metadata: React.FC<{
+  loading: boolean;
+  data: HostQuery;
+  error: ApolloError;
+}> = ({ loading, data, error }) => {
+  const host = data ? data.host : null;
+
+  const hostUrl = host?.hostUrl;
+  const distroId = host?.distroId;
+  const startedBy = host?.startedBy;
+  const provider = host?.provider;
+  const user = host?.user;
+  const lastCommunicationTime = host?.lastCommunicationTime;
+
+  const runningTask = host?.runningTask;
+  const runningTaskId = runningTask?.id;
+  const runningTaskName = runningTask?.name;
+  const sshCommand = `ssh ${user}@${hostUrl}`;
+
+  const taskLink = `${getUiUrl()}/task/${runningTaskId}`;
+  const distroLink = `${getUiUrl()}/distros##${distroId}`;
+  return (
+    <HostMetaDataCard error={error} loading={loading}>
+      <P2>User: {user}</P2>
+      <P2>Host Name: {hostUrl}</P2>
+      <P2>SSH Command: {sshCommand}</P2>
+      <P2>Last Communication: {getDateCopy(lastCommunicationTime)}</P2>
+      <P2>Started By: {startedBy}</P2>
+      <P2>Cloud Provider: {provider}</P2>
+      <P2>
+        Distro:{" "}
+        <StyledLink data-cy="task-distro-link" href={distroLink}>
+          {distroId}
+        </StyledLink>
+      </P2>
+      <P2>
+        Current Task:{" "}
+        <StyledLink data-cy="task-distro-link" href={taskLink}>
+          {runningTaskName}
+        </StyledLink>
+      </P2>
+    </HostMetaDataCard>
+  );
+};
+
+const DATE_FORMAT = "MMM d, yyyy, h:mm:ss aaaa";
+const getDateCopy = (d: Date): string =>
+  d ? format(new Date(d), DATE_FORMAT) : "";
+
+export const P2 = styled.p`
```

Comment:
This looks similar to the P2 in 
https://github.com/evergreen-ci/spruce/blob/329538d5431e16336b3416049d1047f848004efa/src/components/Typography/index.ts#L47

You can probably use that one instead of recreating it here. You should be able to just import it and use it.

Reply:
I put it here because I had to change the css.

Reply:
you can change the css of an existing styled component by doing `const StyledP2 = styled(P2)`

Reply:
the P2 component in Typography seems to be mainly used with the shared MetadataCard component which is styled differently than the host metadata card. I think it makes sense to export this P2 from HostMetadataCard and also export the P2 in typography from the shared MetadataCard component and maybe call it something else like "Row".